app/views/placeholder/home.html.erb

```diff
@@ -11,11 +11,10 @@
 </head>
 <body>
 
-<nav class="navbar navbar-expand-sm navbar-dark" style="background-color: #00ada1">
+<nav class="navbar navbar-expand-sm navbar-dark" style="background-color: #3B7EA1">
   <div class="container">
-  <%= image_tag("LEAD logo header", {:style => "height:60px; padding-right: 20px"}) %>
      
-  <a class="navbar-brand" href="/">Home</a>
+  <a class="navbar-brand" href="/">communities<b>@Cal</b></a>

```

Comment:
I think the LEAD Center logo should be part of the footer, so just leaving this as a placeholder for the actual Communites@Cal logo.

Reply:
Also probably best if we push the nav links to the right.